source/_integrations/template.markdown

```diff
@@ -330,6 +330,36 @@ sensor:
 
 Useful entities to choose might be `sensor.date` which update once per day or `sensor.time` which updates once per minute.
 
+Sometimes it is reasonable to convert an entity-less template that operates with time into one with entities to get your sensor value updated automatically:
```

Comment:
This sentence does not match the example given.
It is not an entity-less template, `binary_sensor.anybody_home` is clearly an entity in the example below.

The problem here is that the state of the template sensor defined is not only depended on changes to entities, but also on time. The Home Assistant state engine does not update/revalute the `value_templates` on time changes, but only on state changes.

Adding in `sensor.time` into the mix, triggers the template, but as a side-effect, the template will be re-evaluated by the state engine each time the time changes (a hell of a lot).

I think we should make this a bit more clear.

Reply:
agree, changed the example and wording